get-involved.php

```diff
@@ -90,6 +90,15 @@
             <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank" rel="noopener">Visit Store</a>
         </div>
     </div>
+
+    <div class="grid">
+        <div class="third">
+            <i class="fab fa-btc"></i>
+            <p>Show your support through coinbase. You will be able to support us via cryptocurrencies easily and securely.</p>
+            <a class="button flat" href="https://commerce.coinbase.com/checkout/d4dbaa95-5a48-42ec-8731-af867e90e4b4">Donate with Crypto</a>
```

Comment:
We avoid the word "donate" since that can imply a tax deduction. We usually say "contribute," "fund," "pay," or similar. In this case, I think we can just say:

```suggestion
            <a class="button flat" href="https://commerce.coinbase.com/checkout/d4dbaa95-5a48-42ec-8731-af867e90e4b4">Pay with Crypto</a>
```